eclipse-collections-code-generator/src/main/resources/api/map/mutableObjectPrimitiveMap.stg

```diff
@@ -111,6 +111,8 @@ public interface MutableObject<name>Map\<K> extends Object<name>Map\<K>
      */
     <type> getIfAbsentPut(K key, <type> value);
 
+    <type> getAndPut(K key, <type> putValue, <type> defaultValue);
```

Comment:
This will need to be implemented as a default method in order to be merged for the 11.1 release.

Reply:
Done.